src/core/test/fuzzing/src/test/scala/Fuzzing.scala

```diff
@@ -143,7 +150,8 @@ trait SerializationFuzzing[S <: PipelineStage with MLWritable] extends FuzzingMe
             testRoundTripHelper(savePath + "/stage", stage, reader, req.fitDF, req.transDF)
           }
           stage.fit(req.fitDF).asInstanceOf[PipelineStage with MLWritable]
-        case stage: Transformer => stage
+        case stage: Transformer =>
+          stage
```

Comment:
This (and other whitespace changes) are for Scalastyle correct?

Reply:
Removed this change